vendor/yosensi/yo-agribox.yaml

```diff
@@ -0,0 +1,90 @@
+YO AgriBox operates over LoRaWAN. The device provides low power consumption and longer battery life. It is high-quality product made in EU.
+
+# Hardware versions (optional, use when you have revisions)
+hardwareVersions:
+  - version: '2.0'
+    numeric: 1
+ 
+
+# Firmware versions (at least one is mandatory)
+firmwareVersions:
+  - # Firmware version
+    version: '3.1.0'
+    numeric: 1
+    hardwareVersions:
+      - '2.0'
+
+    # LoRaWAN Device Profiles per region
+    # Supported regions are EU863-870, US902-928, AU915-928, AS923, CN779-787, EU433, CN470-510, KR920-923, IN865-867,
+    # RU864-870
+    profiles:
+      EU863-870:
+        # Optional identifier of the vendor of the profile. When you specify the vendorID, the profile is loaded from
+        # the vendorID's folder. This allows you to reuse profiles from module or LoRaWAN end device stack vendors.
+        # If vendorID is empty, the current vendor ID is used. In this example, the vendorID is the current vendor ID,
+        # which is verbose.
+        vendorID: 659
+        # Identifier of the profile (lowercase, alphanumeric with dashes, max 36 characters)
+        id: yo-agribox-profile
+        lorawanCertified: true  
+      US902-928:  #dodać profil na US
+        id: yo-agribox-profile-us
+        lorawanCertified: true
+      AU915-928:
+        id: yo-agribox-profile-au
+        lorawanCertified: true
+      
+# Sensors that this device features (optional)
+# Valid values are:
+# 4-20 ma, accelerometer, altitude, analog input, auxiliary, barometer, battery, button, bvoc, co, co2, conductivity,
+# current, digital input, dissolved oxygen, distance, dust, energy, gps, gyroscope, h2s, humidity, iaq, level, light,
+# lightning, link, magnetometer, moisture, motion, no, no2, o3, particulate matter, ph, pir, pm2.5, pm10, potentiometer,
+# power, precipitation, pressure, proximity, pulse count, pulse frequency, radar, rainfall, rssi, smart valve, snr, so2,
+# solar radiation, sound, strain, surface temperature, temperature, tilt, time, tvoc, uv, vapor pressure, velocity,
+# vibration, voltage, water potential, water, weight, wifi ssid, wind direction, wind speed.
+sensors:
+  - moisture
+  - temperature
+  - humidity
+  
+
+
+
+# Additional radios that this device has (optional)
+# Valid values are: ble, nfc, wifi, cellular.
+additionalRadios:
+  - ble
+ 
+
+
+# Dimensions in mm (optional)
+# Use width, height, length and/or diameter
+dimensions:
+  width: 88
+  height: 42
+  length: 64
+
+# Weight in grams (optional)
+weight: 134
+
+
+# IP rating (optional)
+ipCode: IP67
+
+
+# Product and data sheet URLs (optional)
+productURL: https://yosensi.io/products/yo_agri_box/
+dataSheetURL: https://yosensi.io/support/datasheets/YO_AgriBox_datasheet.pdf
+
+
+
+# Photos
+photos:
+  main: YO_AgriBox_01C.png
```

Comment:
please review the name of the file, also should be all in lowercase `yo-agribox-01c` also rename the file to lowercase